primer/src/Primer/Core/Meta.hs

```diff
@@ -63,7 +63,7 @@ newtype ID = ID {unID :: Int}
   deriving newtype (ToJSONKey, FromJSONKey)
   deriving anyclass (NFData)
 
-data Meta a = Meta ID a (Maybe Value)
+data Meta a = Meta !ID a (Maybe Value)
```

Comment:
Could someone independently confirm some time&space measurements for this change?

Reply:
Let's just merge this -- easy enough to revert, and hopefully our benchmark suite will reproduce.